### ISSUE TITLE
Training and Validation Loss (with overfitting)

### ISSUE THREAD

**Issue by AhmadSabbirChowdhury** [AhmadSabbirChowdhury/Cats-Vs-Dogs-Classification-Kaggle-Problem-Set-#2]
![Inaccurate Data 2](https://user-images.githubusercontent.com/45898995/82494128-ea3bfe80-9b0a-11ea-9606-bcd4429af0fd.png)
